compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -639,6 +691,10 @@ class MostSpecificCandidates {
     candidates[VALUE] = sig;
   }
 
+  void setBestOnly(const TypedFnSignature* sig) {
+    candidates[REF] = sig;
```

Comment:
I  am inclined to introduce another enum name, as an alias for REF, for use in this case.